doc/ref_model/chapters/chapter03.md

```diff
@@ -124,24 +124,26 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 <a name="3.2.1.3"></a>
 #### 3.2.1.3 Virtual Storage
 
-A workload can request different types of storage based on data longevity: persistent or ephemeral storage.
+A workload can request different sizes and types of storage based on data longevity: persistent or ephemeral storage.
 Persistent storage outlives the compute instance whereas ephemeral storage is linked to compute instance lifecycle.
 
-There are multiple storage performance requirements such as latency, IOPS and capacity. For example, a workload may require one of its storage device to provide low latency, high IOPS and very large/huge storage capacity (terabytes of data).
+There are multiple storage performance attributes, such as latency, IO/s (Input/Output Operations per second, a.k.a. IOPS), and throughput. For example, a workload may require one of its storage devices to provide low latency, high IO/s and very large/huge storage size (terabytes of data).
```

Comment:
are we defining the terms: **high** IO/s, **very large/huge** storage size

Reply:
It appears that the RM provides quantitative definitions for some of these performance attributes in https://cntt.readthedocs.io/en/latest/ref_model/chapters/chapter04.html#4.2.3  (as you pointed out today). The sentence that begins "For example, ..." is only modified to use consistent terms.